src/utils/PowerBIUtils.js

```diff
@@ -51,7 +51,25 @@ const getScenarioViewReportConfig = (reportsConfig, runTemplateId) => {
     : [];
 };
 
+const _getReportsIdsFromDashboardsViewConfig = (reportsConfig) => reportsConfig.map((report) => report.reportId);
+const _getReportsIdsFromScenarioViewConfig = (reportsConfig) => {
+  if (Array.isArray(reportsConfig)) {
+    return reportsConfig.map((reportConfig) => reportsConfig.map((report) => report.reportId));
```

Comment:
I don't see where is used `reportConfig` parameter ? :thinking: Why do we need a double loop to retrieve `reportId` ?

Reply:
Nice catch, copy/paste mistake :sweat_smile: